docs/stage/download-raw-data.md

```diff
@@ -33,3 +36,19 @@ Run the SQL:
     DELETE FROM download_raw_errors;
     UPDATE publisher_feed SET raw_next_url=NULL;
     
+
+## To only download a limited set of raw data
+
+You may want to do this to avoid processing too much data. 
+Or to debug a problem that occurs with a specific publisher's data, you may want to get data for that publisher only.
+
+### Stop the process early
+
+The process can be forcibly stopped at any point and the database will not be in a bad state. 
+It will contain most work done up to the point you stop it. 
```

Comment:
What will be missing? Is there an intuitive way to know?

Reply:
The statement above says this will contain all work apart from what it's currently doing. That's the same here, and there's no easy way to explain more about "What it's currently doing" that I think makes a difference to the main message this bit is trying to convey.